frontend/wallet/src/Page.re

```diff
@@ -73,24 +73,49 @@ let make = (~message) => {
         ~left="0",
         (),
       )}>
-      <div className=Css.(style([display(`flex), flexDirection(`column)]))>
-        <Header />
-        <Body message={message ++ ";; " ++ settingsInfo} />
+      <div
+        className=Css.(
+          style([
+            display(`flex),
+            flexDirection(`column),
+            justifyContent(`spaceBetween),
+            height(`percent(100.)),
+          ])
+        )>
+        <div>
+          <div
+            className=Css.(style([display(`flex), flexDirection(`column)]))>
+            <Header />
+            <Body message={message ++ ";; " ++ settingsInfo} />
+          </div>
+          <button
```

Comment:
A helper for rendering these buttons may be nice since they mostly follow the same structure.

Reply:
I don't think it's worth it since the "abstracted" version basically has the same amount of syntactic noise.

```
<TestButton message="foo" onClick={handleClick} />
```

vs

```
<button onClick={_e => handleClick()}>
  {ReasonReact.string("foo")}
</button>
```

Were you thinking about this some other way?

Reply:
Well, I wasn't imagining another component, but a helper function for generating jsx.

`button("foo", ~f=...)`